app/src/main/java/io/pslab/communication/ScienceLab.java

```diff
@@ -1588,6 +1798,21 @@ else if ("falling".equals(edgeType2))
         return null;
     }
 
+    /**
+     * Log timestamps of rising/falling edges on one digital input
+     * @param waitingTime Total time to allow the logic analyzer to collect data. This is implemented using a simple sleep routine, so if large delays will be involved, refer to startOneChannelLA() to start the acquisition, and fetchLAChannels() to retrieve data from the hardware after adequate time. The retrieved data is stored in the array self.dchans[0].timestamps.
+     * @param aquireChannel ID1',...,'ID4'
+     * @param triggerChannel ID1',...,'ID4'
+     * @param aquireMode EVERY_SIXTEENTH_RISING_EDGE = 5
```

Comment:
@neel1998 according to the changes requested by @CloudyPadmal , I think here as well all the ID should be mentioned for aquireChannel and triggerChannel params